src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectTags.class.php

```diff
@@ -118,97 +174,88 @@ public function GetTagList()
         $aFoundTags = array();
         $oGlobal = TGlobal::instance();
 
-        if ($oGlobal->UserDataExists('search')) {
-            $sLanguageIsoName = $this->GetLanguageSuffix();
-            $sTagBlackListQueryPart = '';
-            if ($oGlobal->UserDataExists('currentTags')) {
-                $sCurrentTags = $oGlobal->GetUserData('currentTags');
-                $aCurrentTags = explode(',', $sCurrentTags);
-                if (count($aCurrentTags) > 0) {
-                    $sTagBlackList = '';
-                    foreach ($aCurrentTags as $sCurrentTag) {
-                        $sTagBlackList .= "'".MySqlLegacySupport::getInstance()->real_escape_string(trim(strtolower($sCurrentTag)))."',";
-                    }
-                    $sTagBlackList = substr($sTagBlackList, 0, -1);
+        $inputFilter = $this->getInputFilterUtil();
+
+        $searchTerm = $inputFilter->getFilteredInput('q');
```

Comment:
While we're at it, we could restrict this to GET parameters.

Reply:
done